raiden/routing.py

```diff
@@ -185,7 +185,8 @@ def get_best_routes(
 
         log.warning(
             "Request to Pathfinding Service was not successful. "
-            "No routes to the target are found."
+            "No routes to the target are found.",
+            pfs_message=pfs_error_msg,
```

Comment:
Isn't @karlb working on that in https://github.com/raiden-network/raiden/issues/5645?

Reply:
This just adds a bit more logging, Karl is doing something else I think.